iBridgesCli.py

```diff
@@ -71,6 +71,9 @@ def __init__(self,                      # pylint: disable=too-many-arguments
         self.config_file = None
         self.download_finished = None
         self.upload_finished = None
+        self.irods_conn = None
+
+        default_irods_env = os.path.join(str(os.getenv('HOME')), '.irods', 'irods_environment.json')
```

Comment:
You can use a `LocalPath` here:

```python
default_irods_env = utils.path.LocalPath(utils.context.IRODS_DIR, 'irods_environment.json').expanduser()
```

When assigning to a `JsonConfig.filepath`, `expanduser()` is called automatically.

I didn't have a need for it previously, but you could also add `DEFAULT_IRODS_ENV_FILE` to `context.py`:

```python
DEFAULT_IRODS_ENV_FILE = f'{IRODS_DIR}/irods_environment.json'
```

and thus it becomes:

```python
default_irods_env = utils.path.LocalPath(utils.context.DEFAULT_IRODS_ENV_FILE).expanduser()
```

You can update L178-179 to use these constants as well.

Reply:
Perhaps it is even better to use this assignment:

```python
DEFAULT_IBRIDGES_CONF_FILE = path.LocalPath(
    IBRIDGES_DIR, 'ibridges_config.json').expanduser()
DEFAULT_IRODS_ENV_FILE = path.LocalPath(
    IRODS_DIR, 'irods_environment.json').expanduser()
```

Then:

```python
default_irods_env = utils.context.DEFAULT_IRODS_ENV_FILE
```

is possible.